cypress/support/commands.js

```diff
@@ -0,0 +1,149 @@
+import * as testData from '../fixtures/testData.json'
+import Selectors from "../support/selectors.js"
+
```

Comment:
Speaking of commands file it is better to split it like Page Object Model.
I mean mainPageCommands.js, trimCommands.js etc. It is a pattern we use in a project.